checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
Note, that this test case is in the directory for Issue 979 and that it is marked at the top of the file as a test case for Issue 979.  I referenced this issue number after the `@SuppressWarnings` and moved the longer explanation to above the annotation so that the annotation remains on one line.